<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormChatting</class>
 <widget class="QWidget" name="FormChatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gb_sendMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>SendMsg:</string>
   </property>
   <widget class="QLineEdit" name="le_sendMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_cancle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancle</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_send">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_userInfo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>UserInfo:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_uid">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nickname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NickName:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_sign">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_netyear">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NetYear:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_userInfo_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat Records:</string>
   </property>
   <widget class="QTextBrowser" name="tb_records">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_recvMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>RecvMsg:</string>
   </property>
   <widget class="QTextBrowser" name="tb_recvMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
